distros/yamls/java-ebpf-instrumentations.yaml

```diff
@@ -23,4 +23,4 @@ spec:
   runtimeAgent:
     directoryNames:
       # - "{{ODIGOS_AGENTS_DIR}}/java-ebpf"
-    device: 'instrumentation.odigos.io/java-ebpf-enterprise'
+    device: 'instrumentation.odigos.io/agentmount'
```

Comment:
Maybe better name than `aegntmount`? how about `odigos.io/instrumentation` ?

Reply:
or if we need to keep the same prefix I would do
`instrumentation.odigos.io/generic`